app/web/components/MapAnimation.tsx

```diff
@@ -4,21 +4,32 @@ import { LANDING } from "i18n/namespaces";
 import Lottie from "lottie-react";
 import Sentry from "platform/sentry";
 import { useEffect, useState } from "react";
-import { theme } from "theme";
 
 import Alert from "./Alert";
 
 const Wrapper = styled("div")({
   display: "flex",
   justifyContent: "center",
   alignItems: "center",
-  width: "600px",
-  height: "600px",
+  position: "relative",
 
-  [theme.breakpoints.down("md")]: {
-    width: "100%",
-    height: "100%",
-  },
+  width: "100%",
+  height: "100%",
+});
+
+const Attribution = styled("div")({
+  position: "absolute",
+  bottom: 8,
+  right: 10,
+  background: "rgba(255,255,255,0.8)",
+  padding: "2px 8px",
+  borderRadius: 4,
+  fontFamily: "Inter, sans-serif",
```

Comment:
We don't have this font, and we shouldn't add a font just for this. Let's just use ubuntu?